src/webgpu/web_platform/canvas/readbackFromWebGPUCanvas.spec.ts

```diff
@@ -15,6 +15,8 @@ import {
 
 export const g = makeTestGroup(GPUTest);
 
+const kValidSwapchainFormats: GPUTextureFormat[] = ['bgra8unorm', 'rgba8unorm'];
```

Comment:
Let's put this definition in capability_info.ts (just after kRenderableColorTextureFormats).

Also since we don't use the term "swapchain" anymore how about "kCanvasTextureFormats"?

Reply:
Done!